aiohttp_jwt/middleware.py

```diff
@@ -9,22 +10,22 @@
 
 logger = logging.getLogger(__name__)
 
-_request_property = ...
+_request_property: str = ''
```

Comment:
Here should be `Union[Ellipsis, str]` or further checks won't work

Reply:
I added ellipsis but 

`payload = request.get(middleware._request_property)`

Argument 1 to "get" of "Mapping" has incompatible type "Union[ellipsis, str]"; expected "str"mypy(error)